### PR TITLE
New article: 2-factor authentication on Cloud

### PR DESCRIPTION
**Pull request by jonat123**
Added a new article for 2-factor authentication on Umbraco Cloud.

### CODE REVIEW

**Review by Fjellvang — COMMENTED** [review]
Maybe we have to be carefull with the QR code in Auth-app.png ?

I think its good to replace that same screenshot with a enrollement picture from a dummyaccount or dev etc?

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+versionFrom: 8.0.0
+versionTo: 10.0.0
+meta.Title: "2-factor authentication on Umbraco Cloud"
+meta.Description: "This article shows you how you can enable 2-factor authentication for when you log in to the Umbraco Cloud Portal."
+---
+
+# 2-factor authentication on Umbraco Cloud
+
+On Umbraco Cloud, you can add 2-factor authentication for your Umbraco Cloud account using either email, phone or a authenticator app when logging in to the Umbraco Cloud Portal.
```
Comment (jonat123) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:10]:
```suggestion
On Umbraco Cloud, you can add 2-factor authentication for your Umbraco Cloud account using either email, phone, or an authenticator app when logging in to the Umbraco Cloud Portal.
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Enabeling 2-factor authentication
```
Comment (jonat123) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:12]:
```suggestion
## Enabling 2-factor authentication
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+To enable 2-factor-authentication follow these steps:
+
+1. Go to your profile on Umbraco Cloud
+2. In the profile settings section go to Edit profile
+3. Choose the desired 2-factor authentication
+4. Follow the steps shown to enable 2-factor authentication.
+
+### Email authentication
+
+With email authentication you will get an email with a code that you need to enter when you log in to the Umbraco Cloud portal.
```
Comment (jonat123) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:25]:
```suggestion
With email authentication, you will get an email with a code that you need to enter when you log in to the Umbraco Cloud portal.
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+![Email authentication](images/email-auth.png)
+
+### Authenticator App
+
+With authenticator app, you need to use a authenticator app when you log in to the Umbraco Cloud Portal.
+
+You can use the Microsoft authenticator app for both iOS and android or any other authenticator app of your choice.
```
Comment (jonat123) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:33]:
```suggestion
With the authenticator app, you need to use an authenticator app when you log in to the Umbraco Cloud Portal.

You can use the Microsoft Authenticator app for both iOS and android or any other authenticator app of your choice.
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+![Authenticator app](images/auth-app.png)
+
+### Phone Authentication
+
+With the phone authenticator, when you log in to the Umbraco Cloud portal, you can choose to either receive a SMS with a code to enter when login in or a call to log you in.
```
Comment (jonat123) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:39]:
```suggestion
With the phone authenticator, when you log in to the Umbraco Cloud portal, you can choose to either receive an SMS with a code to enter when login in or a call to log you in.
```

**Comment by sitereactor** [comment]
I think it looks good, but maybe end this article with a "Logging in with 2FA" section where you explain/highlight what the login flow will look like with 2FA in place. And just use one of the MFA methods as an example.
Then we'll have both the enrollment and login parts covered.

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+![Email authentication](images/email-auth.png)
+
+### Authenticator App
+
+With the authenticator app, you need to use an authenticator app when you log in to the Umbraco Cloud Portal.
+
+You can use the Microsoft Authenticator app for both iOS and android or any other authenticator app of your choice.
+
+![Authenticator app](images/auth-app.png)
```
Comment (Fjellvang) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:35]: as mentioned in comment, ensure the image is not a valid QR code for your production account

**Comment by jonat123** [comment]
@Fjellvang that makes total sense, i did not think about it when taking the images 😅

I'll make sure that the QR code is just a dummy one that does not work!

**Review by Fjellvang — CHANGES_REQUESTED** [review]

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+meta.Description: "This article shows you how you can enable 2-factor authentication for when you log in to the Umbraco Cloud Portal."
+---
+
+# 2-factor authentication on Umbraco Cloud
+
+On Umbraco Cloud, you can add 2-factor authentication for your Umbraco Cloud account using either email, phone, or an authenticator app when logging in to the Umbraco Cloud Portal.
+
+## Enabling 2-factor authentication
+
+2-factor authentication can be enabled when editing your Umbraco Cloud profile
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:14]:
```suggestion
2-factor authentication can be enabled when editing your Umbraco Cloud profile.
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+To enable 2-factor-authentication follow these steps:
+
+1. Go to your profile on Umbraco Cloud
+2. In the profile settings section go to Edit profile
+3. Choose the desired 2-factor authentication
+4. Follow the steps shown to enable 2-factor authentication.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:21]:
```suggestion
1. Go to your profile on Umbraco Cloud.
2. Choose **Edit Profile** in the Profile Settings section.
3. Choose the desired 2-factor authentication.
4. Follow the steps shown to enable 2-factor authentication.
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+### Email authentication
+
+With email authentication, you will get an email with a code that you need to enter when you log in to the Umbraco Cloud portal.
+
+![Email authentication](images/email-auth.png)
+
+### Authenticator App
+
+With the authenticator app, you need to use an authenticator app when you log in to the Umbraco Cloud Portal.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:31]:
```suggestion
You have the option to use an Authenticator App when logging in to the Umbraco Cloud Portal.
```

File: Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+You can use the Microsoft Authenticator app for both iOS and android or any other authenticator app of your choice.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/2-factor-authentication-on-cloud/index.md:33]:
```suggestion
You can use the Microsoft Authenticator App for both iOS and Android or any other authenticator app of your choice.
```

**Review by sofietoft — COMMENTED** [review]
Article looks great @jonat123 ! 💪 

I've made a few comments here and there.
Let me know what you think!

**Comment by jonat123** [comment]
Thanks for the reviews!

The QR code has been replaced with one that just leads back to our.umbraco.com instead :) 

@sitereactor I think it is fine to just show how to enable the 2-factor authentication, and not more than that, once it is enabled I think the steps are pretty clear in what is shown on the Umbraco Cloud portal :)

**Comment by jonat123** [comment]
I am merging this in now :D

**Comment by sitereactor** [comment]
Alright 👍

**Pull request merged**